### PR TITLE
[T4A3][W09-A1] Low Yong Siang

### PR DESCRIPTION
**Pull request by YongSiang94**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Contact.java:5]: No header comment.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof Address // instanceof handles nulls
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/data/person/Contact.java:39]: Is it okay to leave it as `Address`?

**Comment by jeffryhartanto** [comment]
@YongSiang94 
Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/person/Contact.java (hunk 2 above)
Comment (YongSiang94) [reply src/seedu/addressbook/data/person/Contact.java:39]: Sorry. I didn't change it to Contact.

**Pull request closed without merging**